nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/test/java/org/apache/nifi/reporting/TestSiteToSiteProvenanceReportingTask.java

```diff
@@ -36,6 +36,7 @@
 import org.apache.nifi.util.MockFlowFile;
 import org.apache.nifi.util.MockPropertyValue;
 import org.junit.Assert;
+import org.junit.Ignore;
```

Comment:
This import is not used and raises a checkstyle violation. I can take care of it while merging but just want to double check if there is a test you wanted to add (even if ignored in the build)?

Reply:
This is a leftover when I struggled with making local contrib-check to pass. Please remove this import statement. Thanks!